doc/lua_api.txt

```diff
@@ -580,25 +580,25 @@ index of the pixel to use.
 When registering a node, set the item definition's `palette` field to
 a texture. You can also use texture modifiers.
 The node's color depends on its `param2`, so you also must set an
-appropriate `drawtype`:
+appropriate `paramtype2`:
 
-* `drawtype = "color"` for nodes which use their full `param2` for
+* `paramtype2 = "color"` for nodes which use their full `param2` for
   palette indexing. These nodes can have 256 different colors.
   The palette should contain 256 pixels.
-* `drawtype = "colorwallmounted"` for nodes which use the first
+* `paramtype2 = "colorwallmounted"` for nodes which use the first
   five bits (most significant) of `param2` for palette indexing.
   The remaining three bits are describing rotation, as in `wallmounted`
-  draw type. Division by 8 yields the palette index (without stretching the
+  drawtype. Division by 8 yields the palette index (without stretching the
```

Comment:
Same here

Reply:
You're right. These two are indeed values for `paramtype2`.

Reply:
Thanks will fix.